web-ui/src/components/volunteer/VolunteerEvents.jsx

```diff
@@ -0,0 +1,489 @@
+import { format } from 'date-fns';
```

Comment:
This bit of UI renders a table of volunteering events and supports all the CRUD operations. It renders somewhat different when `onlyMe` is true which is the case when this is used on the Profile page of the current user.